<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="Page">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>761</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="p_intro">
     <widget class="QPushButton" name="btn_join">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>회원가입</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_login">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>로그인</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="p_main">
     <widget class="QToolBox" name="Menu">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>731</width>
        <height>671</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(221, 227, 240)</string>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="p_stock">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>731</width>
         <height>567</height>
        </rect>
       </property>
       <attribute name="label">
        <string>주문 관리</string>
       </attribute>
       <widget class="QTableWidget" name="order_table">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>451</width>
          <height>121</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>품목</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>수량</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>상태</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>재료</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="p_stock_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>731</width>
         <height>567</height>
        </rect>
       </property>
       <attribute name="label">
        <string>재고 관리</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="page_3">
       <attribute name="label">
        <string>문의 관리</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>731</width>
         <height>567</height>
        </rect>
       </property>
       <attribute name="label">
        <string>상품 관리</string>
       </attribute>
       <widget class="QTableWidget" name="product_table">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>281</width>
          <height>541</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 255, 255);</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <column>
         <property name="text">
          <string>품목</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>가격</string>
         </property>
        </column>
       </widget>
       <widget class="QTableWidget" name="product_ingredient">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>10</y>
          <width>201</width>
          <height>541</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 255, 255);</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <column>
         <property name="text">
          <string>재료</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>분량</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="p_join">
     <widget class="QPushButton" name="btn_check">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>130</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>중복확인</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="in_name">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="in_id">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="in_pass">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>270</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="in_pass_">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>340</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>100</y>
        <width>331</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt; 중복이면 Q메시지박스가 중복이라고 해야함</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_newuser">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>가입</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>78</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>이름</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>아이디</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>비밀번호</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>비밀번호 확인</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
